backend/src/main/java/me/performancereservation/domain/refund/Refund.java

```diff
@@ -22,14 +22,15 @@ public class Refund extends BaseEntity {
 
     private String bank; // 환불 받을 은행
 
+    @Setter
     @Enumerated(EnumType.STRING)
     private RefundStatus status; // 환불대기, 환불완료 상태 표시
 
-    public void setStatus(RefundStatus status) {
-        // 유효성검사 추가
-
-        this.status = status;
-    }
+//    public void setStatus(RefundStatus status) {
+//        // 유효성검사 추가
+//
+//        this.status = status;
+//    }
```

Comment:
```suggestion
    public void confirm() {
       if (this.status == RefundStatus.CONFIRMED) {
          throw ErrorCode. INVALID_REFUND_STATUS.domainException("이미 환불이 완료된 상태입니다.");
        }

        this.status = RefundStatus.CONFIRMED;
}
```

setter 대신에 Refund 도메인에 맞는 행위 메서드를 써보시는건 어떠신가요?

Reply:
오 좋습니다 반영하겠습니다!